packages/web-components/fast-foundation/src/text-field/text-field.template.ts

```diff
@@ -2,6 +2,18 @@ import { html, ref, slotted } from "@microsoft/fast-element";
 import { endTemplate, startTemplate } from "../patterns";
 import { TextField } from "./text-field";
 
+/***
+ * A simple whitespace filter for the template
+ * @public
+ */
+export function whitespaceFilter(value, index, array) {
+    console.log(value, value.nodeValue);
```

Comment:
Please remove the `console.log` 😄

Reply:
ahah oops, luckily it was still a draft :P, removed.